conda/base/context.py

```diff
@@ -531,6 +533,9 @@ def get_help_dict():
             this setting, specifying that certain files should never be soft-linked (see the
             no_link option in the build recipe documentation).
             """),
+        # 'alt_features': dals("""
```

Comment:
Is this intentionally commented out?

Reply:
Yes, @kalefranz in https://github.com/conda/conda/pull/6296#discussion_r154113746:
> Also, I think it's best--for the time being--to
> 1. leave this configuration gate undocumented, which means removing the alt_features entry here, and instead adding the key to the list around line 469 (feel free to leave the description as a comment there), and